<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>278</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search for:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="searchText">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>In:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="inSummaries">
        <property name="text">
         <string>Summaries</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="inDescriptions">
        <property name="text">
         <string>Descriptions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="inCategories">
        <property name="text">
         <string>Categories</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="inLocations">
        <property name="text">
         <string>Locations</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Between:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
     <item>
      <widget class="KDateComboBox" name="startDate">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>and</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KDateComboBox" name="endDate">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="includeTodosWithoutDueDate">
     <property name="text">
      <string>Include To-Dos without a due date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Located in:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="locatedInAnyCollection">
        <property name="text">
         <string>any folder</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="locatedInSpecificCollection">
        <property name="text">
         <string>only in folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Akonadi::CollectionComboBox" name="collectionCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>Akonadi::CollectionComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">AkonadiWidgets/collectioncombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>locatedInSpecificCollection</sender>
   <signal>toggled(bool)</signal>
   <receiver>collectionCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
